### ISSUE TITLE
Pull latest version from devop tool changes.

### ISSUE THREAD

**Issue by sima-zhu** [Azure/azure-sdk-tools#2347]
**Current status:**
We currently manually update the tool version every time we updates the tool.

**Expect to have**:
The script will pull the latest tool version from devops to avoid the second PR for updating version.

**Things to take care:**
We are lack of the ability to test scripts and tool changes PR. We have to test the bundle scripts with latest tool changes to make sure we did not introduce breaking changes which would probably fail the pipelines.

**Comment by weshaggard** [comment]
>The script will pull the latest tool version from devops to avoid the second PR for updating version.

I don't think we are trying to avoid a second PR. We want the PR to still validate the changes but we want that PR to automatically be generated after we publish a new version of a tool.

**Comment by benbp** [comment]
> > The script will pull the latest tool version from devops to avoid the second PR for updating version.
> 
> I don't think we are trying to avoid a second PR. We want the PR to still validate the changes but we want that PR to automatically be generated after we publish a new version of a tool.

Agreed this will let us capture specific tool versions used in historical builds/releases to reproduce them if needed.
